elements/campaign-display/components/campaign-display-root.js

```diff
@@ -1,4 +1,5 @@
 import React, { PropTypes, Component } from 'react';
+import createFragment from 'react-addons-create-fragment'; // http://facebook.github.io/react/docs/create-fragment.html
```

Comment:
Remove the comment here, I'd finding this documentation is trivial and first-party to react.